packages/bundler-plugin-core/src/debug-id-upload.ts

```diff
@@ -298,10 +298,28 @@ async function determineSourceMapPathFromBundle(
   const sourceMappingUrlMatch = bundleSource.match(/^\s*\/\/# sourceMappingURL=(.*)$/m);
   if (sourceMappingUrlMatch) {
     const sourceMappingUrl = path.normalize(sourceMappingUrlMatch[1] as string);
-    if (path.isAbsolute(sourceMappingUrl)) {
-      return sourceMappingUrl;
+
+    let isSupportedUrl;
+    try {
+      const url = new URL(sourceMappingUrl);
+      isSupportedUrl = url.protocol === "file:";
+    } catch {
+      isSupportedUrl = false;
+    }
+
+    let absoluteSourceMapPath;
+    if (isSupportedUrl || path.isAbsolute(sourceMappingUrl)) {
+      absoluteSourceMapPath = sourceMappingUrl;
     } else {
-      return path.join(path.dirname(bundlePath), sourceMappingUrl);
+      absoluteSourceMapPath = path.join(path.dirname(bundlePath), sourceMappingUrl);
+    }
```

Comment:
m: We reach this branch even if `isSupportedUrl === false`. Is this by intentional?

Reply:
You're right! Fixed in 9c5b99fdd3a9b8273682d8d113528fb3eedcd032